CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- cosmwasm-std: Implement `Not` for `Uint{64,128,256}` ([#1799]).
```

Comment:
```suggestion
- cosmwasm-std: Implement `Not` for `Uint{64,128,256}`/`Int{64,128,256}` ([#1799]).
```

Reply:
The signed ones had it already, I only added the test there

Reply:
Ah, okay, I missed that